docs/docs/recipes/integrate-logto/express.mdx

```diff
@@ -166,6 +166,25 @@ app.get('/protected', requireAuth, (req, res) => {
 });
 ```
 
+## Get access token
+
+If you want to get the access token, set `getAccessToken` to `true`:
+
+```ts
+app.get(
+  '/fetch-access-token',
+  withLogto({
+    ...config,
+    // Fetch access token from remote, this may slowdown the response time,
+    // you can also add "resource" if needed.
+    getAccessToken: true,
+  }),
+  (request, response) => {
+    response.json(request.user);
```

Comment:
will it be better to demonstrate how to retrieve the Access Token here?

Reply:
updated